.github/CONTRIBUTING.md

```diff
@@ -18,7 +18,19 @@ There are many ways to contribute to Codecademy Docs:
 - Give feedback in the [Typeform Survey](https://codecademyready.typeform.com/to/hzVIWDgz).
 - Join the [#CodecademyDocs](https://twitter.com/search?q=%23CodecademyDocs&src=typed_query&f=live) discussion on Twitter.
 
-If you're new to Codecademy Docs and contributing for the first time, it is recommended that you visit the [Issues](https://github.com/Codecademy/docs/issues) section and ask to be assigned to an open issue that interests you. Otherwise, feel free to submit a [PR](https://www.codecademy.com/resources/docs/git/pull-requests) by creating a new [branch](https://www.codecademy.com/resources/docs/general/git/branch) in your fork to create a new entry or edit an existing one.
+If you're new to Codecademy Docs and contributing for the first time, it is recommended that you visit the [Issues](https://github.com/Codecademy/docs/issues) section and ask to be assigned to an [open issue](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md#how-do-i-claim-an-issue) that interests you. Otherwise, feel free to submit a [PR](https://www.codecademy.com/resources/docs/git/pull-requests) by creating a new [branch](https://www.codecademy.com/resources/docs/general/git/branch) in your fork to create a new entry or edit an existing one.
+
+## How do I claim an Issue?
+
+Becoming an assignee to an issue follows this workflow:
+
+- Comment on an issue thread that you're interested in taking the issue, e.g., [Issue #1026](https://github.com/Codecademy/docs/issues/1026).
+- Await a maintainer's response to be assigned.
+  - If assigned, start working on a PR.
+  - If not assigned, continue steps 1 and 2 until assigned an issue.
+- If a PR has not been opened in 2 weeks for the issue, the issue may be reassigned to another contributor. Maintainers will do the best to notify contributors by tagging their GitHub handle along the issue thread.
```

Comment:
```suggestion
1. Comment on a thread for an issue that you're interested in being assigned to (e.g., [Issue #1026](https://github.com/Codecademy/docs/issues/1026)).
2. Await a maintainer's response to be assigned.
  - If assigned, start working on a PR.
  - If not assigned, continue steps 1 and 2 until assigned an issue.
3. If a PR has not been opened in 2 weeks for the issue, the issue may be reassigned to another contributor. Maintainers will do their best to notify contributors by tagging their GitHub handle along the issue thread every 5-7 days.
```

A few notes with this suggestion:

- I think it makes sense to turn this into a numbered list, especially since the one sub-point refers to numbered steps.
- At the end of the third step, I think we should mention how frequently we may notify contributors; 5-7 days seems like a fair frequency, no?

Reply:
seems fair to me too!